### ISSUE TITLE
Colour combination

### ISSUE THREAD

**Issue by officialgupta** [rodohanna/OpenCodenames#3]
For the red team spies, the red cards on the red background is not ideal

**Comment by rodohanna** [comment]
I agree that it's not the best. The one part I do like is that it obviously communicates which team you are on. Do you have any recommendations on how to keep which team each spy is on as obvious as possible and fix the red background issue?

**Comment by officialgupta** [comment]
I haven't formally considered it, but my initial thoughts were that the team background colour could be changed to a more subdued or different shade so the cards stand out. And / or the background could be patterned to distinguish it from the cards [like this](https://css-tricks.com/stripes-css/).

**Comment by rodohanna** [comment]
Thanks for the ideas! I couldn't find a satisfactory solution for now. I'd like to take some more time to experiment with the UI.

For now, I have added a 1px solid black border around each of the cards. I believe this is a good temporary fix to address the contrast issue.
